lib/components/Checkbox/Checkbox.tsx

```diff
@@ -38,7 +50,14 @@ export const Checkbox = ({ name, disabled, value, checked, onChange, label, aria
         checked={checked}
         onChange={onChange}
         aria-label={label ? undefined : ariaLabel}
-        className="ds-peer ds-relative ds-size-5 ds-min-h-5 ds-min-w-5 ds-appearance-none ds-rounded-none ds-bg-white"
+        className={cx(
+          'ds-peer ds-relative ds-size-5 ds-min-h-5 ds-min-w-5 ds-appearance-none ds-rounded-none ds-bg-white',
+          {
+            'ds-border ds-border-accent': variant === 'bordered',
+            'ds-border ds-border-inactive-gray': variant === 'bordered' && disabled,
+            'ds-border-0': variant === 'default',
+          },
+        )}
```

Comment:
This is already pretty nice, but would it be nicer to separate the border classes little more? I think it makes it more explicit what that block does.

```tsx
  const borderVariantClassnames = {
    'ds-border ds-border-accent': variant === 'bordered',
    'ds-border ds-border-inactive-gray': variant === 'bordered' && disabled,
    'ds-border-0': variant === 'default',
  };

// ...
        className={cx(
          'ds-peer ds-relative ds-size-5 ds-min-h-5 ds-min-w-5 ds-appearance-none ds-rounded-none ds-bg-white',
          borderVariantClassnames
        )}
// ...


```

Reply:
Great idea, makes the code more clear!